src/win/pipe.c

```diff
@@ -872,6 +894,14 @@ int uv_pipe_connect2(uv_connect_t* req,
     }
```

Comment:
Why is there an error for exceeding 256, when the underlying API appears to have no such value where truncation occurs (https://github.com/libuv/libuv/blob/de86ae6e4a8a86c063bee916aedf08270d4aa6e0/src/win/pipe.c#L2487-L2488)

Reply:
Or rather, why is the error enforced on the utf8 representation of the name, when that does not correspond to the utf16 encoding size that would generate an error later?

Reply:
I probably should've added that as a comment but I took that from MSDN, see https://github.com/libuv/libuv/issues/4036#issue-1740238471.

Reply:
I think we should just remove that check, since the underlying function will already assert that in whatever the relevant version of Windows there is, and will do so in UTF8 characters

Reply:
I don't disagree. I'll rework this today or tomorrow.

Reply:
Any success with that?